packages/paypal-commerce-integration/src/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -223,6 +323,14 @@ export default class PayPalCommercePaymentStrategy implements PaymentStrategy {
         }
     }
 
+    private async createOrder() {
+        const { getFieldsValues } = this.paypalcommerce || {};
+
+        return this.paypalCommerceIntegrationService.createOrder('paypalcommercecheckout', {
+            shouldSaveInstrument: getFieldsValues?.().shouldSaveInstrument || false,
```

Comment:
Same here regarding object destructuring.

Reply:
It is not necessary here

Reply:
What happens if developer provide `string` instead of `function` in `getFieldsValues`?

Reply:
Should we throw an error if `getFieldsValues` data type is not correct?

Reply:
Updated this part of the code